prompt_testing/prompts/current.txt

```diff
@@ -0,0 +1,12 @@
+You are an expert in {arch} assembly code and {language}, helping users of the
```

Comment:
I know this is our existing prompt but these variables here are not being expanded (I think?). And there's a to-do to not do this.

Reply:
Done